package.json

```diff
@@ -83,5 +84,6 @@
 		"typescript": "^4.6.2",
 		"unified": "^10.0.0",
 		"unified-message-control": "^4.0.0"
-	}
+	},
+	"packageManager": "yarn@3.2.3"
```

Comment:
```suggestion
	"packageManager": "yarn@"3.2.3+sha224.953c8233f7a92884eee2de69a1b92d1f2ec1655e66d08071ba9a02fa"
```

Reply:
what's the purpose of this hash?

Reply:
```console
$ corepack yarn@3.2.1+sha224.deadbeef
Internal Error: Mismatch hashes. Expected deadbeef, got 953c8233f7a92884eee2de69a1b92d1f2ec1655e66d08071ba9a02fa
```

Reply:
so it's an extra security layer to prevent accidentally downloading an attacker's yarn or a botched yarn release from the facebook team?

Reply:
> botched yarn release from the facebook team?

That would be extremely messed up as Facebook/Meta should have no control at all over that, they were only behind Yarn 1.x: https://yarnpkg.com/getting-started/qa#is-yarn-operated-by-facebook

> so it's an extra security layer to prevent accidentally downloading an attacker's yarn

Yes, it's a validation in case HTTPS is not enough / if the remote server was compromised.